.buildkite/steps/script.sh

```diff
@@ -16,10 +16,10 @@ pip freeze
 
 echo "+++ running tests"
 exitCode=$?
-py.test -ra --cov=stellargraph tests/ --doctest-modules --doctest-modules --cov-report=term-missing -p no:cacheprovider --junitxml="./${junit_file}" || exitCode=$?
+py.test -ra --cov=stellargraph tests/ --doctest-modules --cov-report=xml -p no:cacheprovider --junitxml="./${junit_file}" || exitCode=$
 
-echo "--- uploading coveralls"
-coveralls
+echo "--- :coverage::codecov::arrow_up: uploading coverage to codecov.io"
+bash <(curl https://codecov.io/bash)
```

Comment:
Do you think it's potentially unsafe pulling in a script from the internet to execute on our build machines? We could commit the script to the repo and execute it when needed?

Reply:
Codecov claim that since this is https it's no different to trusting things like npm (we already do this for neon...) . if we committed then we'd be manually updating when codecov makes changes.

Reply:
I think the biggest downside here is the download sometimes seems to fail/timeout (at least it did on neon); but having to remember to manually update seems somewhat worse.

Reply:
Yeah makes sense. Perhaps when we solve adding https://github.com/stellargraph/stellargraph/pull/674 we can have the script pulled down in the docker image so we don't have to worry about network connections